.github/workflows/automation.yml

```
@@ -51,14 +51,21 @@
+      
   build-scala-mantic:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
     - name: scala-mantic
-      run: docker build . --file ./scala/Dockerfile.mantic --tag scala-mantic:$(date +%s)
+      run: docker build . --file ./scala/build/Dockerfile.mantic --tag "scala-mantic:$(date +%s)" 
+  build-scala-maintenance:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)